core/client/hdfs3/pom.xml

```diff
@@ -0,0 +1,57 @@
+<!--
+
+    The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+    (the "License"). You may not use this work except in compliance with the License, which is
+    available at www.apache.org/licenses/LICENSE-2.0
+
+    This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+    either express or implied, as more fully set forth in the License.
+
+    See the NOTICE file distributed with this work for information regarding copyright ownership.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.alluxio</groupId>
+        <artifactId>alluxio-core-client</artifactId>
+        <version>2.10.0-SNAPSHOT</version>
```

Comment:
@Xenorith do we need to update the script for this new pom file?

Reply:
no need, the script looks for the version string in all pom.xml files:
```
function update_poms() {
    find . -name pom.xml | xargs -t -n 1 perl -pi -e "s/${1}/${2}/g"
}
```